deployment/cloudformation/majorkirby.py

```diff
@@ -0,0 +1,497 @@
+import sys
```

Comment:
Might want to put a note in here to remove this file once major kirby is in its own repo and installable via pip